tabbycat/draw/urls_admin.py

```diff
@@ -29,9 +29,12 @@
             name='draw-regenerate'),
 
         # Email
-        path('email/',
+        path('email-adjudicators',
             views.EmailAdjudicatorAssignmentsView.as_view(),
-            name='draw-email'),
+            name='draw-adj-email'),
+        path('email-debaters',
+            views.EmailAdjudicatorAssignmentsView.as_view(),
```

Comment:
Was this meant to be `EmailTeamAssignmentsView`?

Reply:
#995.